src/lib/login/userExists.js

```diff
@@ -19,3 +19,14 @@ export const userExists = async (mnemonics): Promise<any> => {
 
   return { exists, fullName }
 }
+
+export const userExists2 = async ({ mnemonics, privateKey, email, mobile }): Promise<any> => {
```

Comment:
Let's use the single function 

export const userExists = async (mnemonics, extra = null): Promise<boolean> {
  const { privateKey, email, mobile } = extra || {}

  if (extra) 
    // userExists2 logic
  else 
    // userExists logic
}
```

the same for the API.userExists

Reply:
fixed